modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/util/ObjectEntryUtil.java

```diff
@@ -103,22 +102,15 @@ public static Object getValue(
 		else if (objectField.compareBusinessType(
 					ObjectFieldConstants.BUSINESS_TYPE_DATE)) {
 
-			Format dateFormat = FastDateFormatFactoryUtil.getDate(
-				DateFormat.DEFAULT, locale, timeZone);
-
-			return dateFormat.format(
-				DateUtil.parseDate("yyyy-MM-dd", value.toString(), locale));
+			return DateUtil.parseDate("yyyy-MM-dd", value.toString(), locale);
 		}
 		else if (objectField.compareBusinessType(
 					ObjectFieldConstants.BUSINESS_TYPE_DATE_TIME)) {
 
-			Format dateFormat = FastDateFormatFactoryUtil.getDateTime(
-				DateFormat.DEFAULT, DateFormat.DEFAULT, locale, timeZone);
+			DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(
+				ObjectFieldUtil.getDateTimePattern(value.toString()));
 
-			return dateFormat.format(
-				DateUtil.parseDate(
-					ObjectFieldUtil.getDateTimePattern(value.toString()),
-					value.toString(), locale));
+			return dateTimeFormatter.parse(value.toString());
```

Comment:
@victorg1991 I'm testing this code by adding a Collection Display (**Style Display**: Table) mapped to a custom Object with a DateTime field and I'm getting this error: 
```
java.lang.ClassCastException: java.time.format.Parsed cannot be cast to java.io.Serializable
        at com.liferay.object.web.internal.info.item.renderer.ObjectEntryRowInfoItemRenderer._getValues(ObjectEntryRowInfoItemRenderer.java:164) ~[?:?]
        at com.liferay.object.web.internal.info.item.renderer.ObjectEntryRowInfoItemRenderer.render(ObjectEntryRowInfoItemRenderer.java:85) ~[?:?]
        at com.liferay.object.web.internal.info.item.renderer.ObjectEntryRowInfoItemRenderer.render(ObjectEntryRowInfoItemRenderer.java:45) ~[?:?]
```